units/host_drawdown_test.go

```diff
@@ -42,7 +43,10 @@ func numHostsTerminated(ctx context.Context, env evergreen.Environment, drawdown
 //
 
 func TestTerminatingHosts(t *testing.T) {
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	ctx = testutil.TestSpan(ctx, t)
+
 	env := evergreen.GetEnvironment()
 
 	t.Run("SimpleTerminationTest", func(t *testing.T) {
```

Comment:
we could make another span for the subtest. 
It seems like a waste, though, since it's just this one subtest.

Reply:
It seems like the subtest span will include the subtest name, and the parent span wouldn't, correct? Also, if someone updates this test in the future it will be more clear that a new subtest should have its own span.

Reply:
ah, okay. That makes sense
done